src/utils/DataUtils.js

```diff
@@ -34,3 +34,9 @@ export const getDateRangeSearchTerm = (
   propertyTypeId,
   searchString
 ) => `${SEARCH_PREFIX}.${propertyTypeId}:${searchString}`;
+
+// https://github.com/immutable-js/immutable-js/wiki/Predicates#pick--omit
+export const keyIn = (keys) => {
```

Comment:
I don't see this anywhere in the PR. What's this for?

Reply:
Oh. At some point i was using this to help 'pluck' subsets.

```
const originalMap = Map({ a: 1, b: 2, c: 3, d: 4 });
const plucked = originalMap.filter(keyIn(['a', 'b'])); // ==> Map({ a: 1, b: 2 });
const omitted = originalMap.filterNot(keyIn(['a', 'b']));  // ==> Map({ c: 1, d: 2 });
```

 But then I rewrote the component to not need this at all. Will remove.